weaver/wps_restapi/swagger_definitions.py

```diff
@@ -3779,9 +3831,11 @@ class CWLRequirements(OneOfKeywordSchema):
 class CWLHintsMap(AnyOfKeywordSchema, PermissiveMappingSchema):
     _any_of = [
         BuiltinRequirementMap(missing=drop),
+        CudaRequirementMap(missing=drop),
         DockerRequirementMap(missing=drop),
         DockerGpuRequirementMap(missing=drop),
         InitialWorkDirRequirementMap(missing=drop),
+        NetworkAccessRequirementMap(missing=drop),
```

Comment:
Corresponding items for the "List" representation are missing (see other hints schemas for examples).